content/altinn-studio/guides/monitor-and-instrument/_index.nb.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Monitorer og instumenter
+linktitle: Monitorer og instumenter
+description: Kom i gang med verktøyene for instrumentering og monitorering i Altinn
+toc: true
+weight: 990
+---
+
+{{% notice info %}}
+Denne siden refererer til konfigurasjon når [OpenTelemetry (OTel)](https://opentelemetry.io/) er i bruk i appen fra v8 og nyere.
+Det eksisterende Application Insights SDK oppsettet er utgått, og vil fjernes i neste hovedversjon av Altinn.App biblioteker
+{{% /notice %}}
+
+## Konfigurering av en app
+
+Det nye monitorerings- og instrumenteringsoppsettet basert på OpenTelemetry kan aktiveres ved å sette 
+`UseOpenTelemetry` til `true` i `appsettings.json` eller ekvivalent.
+
+{{< highlight json "linenos=false,hl_lines=3" >}}
+{
+  "AppSettings": {
+    "UseOpenTelemetry": true
+  }
+}
+{{< / highlight >}}
+
+Når appen kjøres med denne instillingen så vil Altinn.App biblioteket sende telemetri til localtest ved lokal kjøring
+og til Azure Monitor ved kjøring i et miljø.
+
+[Les mer om konfigurasjonsmuligheter på referansesiden for konfigurasjon](/nb/altinn-studio/reference/monitoring/configuration).
+
+## Egendefinert instrumentering
+
+Vi illustrerer egendefinert instrumentering med et eksempel. I `Program.cs` legger vi til en simpel `IHostedService` implementasjon
```

Comment:
Using ` to highlight words result in the copy icon which breaks the flow for the reader. Unless a path you believe the reader will need to copy maybe _italics_ could be an option instead ?
![image](https://github.com/Altinn/altinn-studio-docs/assets/47737608/1606b8ad-55ad-4249-9781-e599d71c231e)